pal/src/map/common.cpp

```diff
@@ -49,7 +49,7 @@ INT W32toUnixAccessControl( IN DWORD flProtect )
         MemAccessControl = PROT_EXEC | PROT_READ | PROT_WRITE;
         break;
     case PAGE_EXECUTE :
-        MemAccessControl = PROT_EXEC;
+        MemAccessControl = PROT_EXEC | PROT_READ; // WinAPI PAGE_EXECUTE also implies readable
```

Comment:
@curtisman @digitalinfinity this looks to me a workaround rather than a fix. See `PAGE_EXECUTE_READ` below. If our code needs to read the memory that has `PAGE_EXECUTE` rights, shouldn't we better set `PAGE_EXECUTE_READ` instead ? I see on Windows, `PAGE_EXECUTE` gives a limited READ capabilities? But adding `PROT_READ` doesn't look to me as a similar behavior.